extensions/azurePublish/src/components/util.tsx

```diff
@@ -0,0 +1,100 @@
+import { getTenants, getARMTokenForTenant } from '@bfc/extension-client';

```

Comment:
this file is not formatted correctly. e.g., the copyright missing on top, spacings are off, etc.